src/bundle/Resources/public/js/scripts/core/storage.js

```diff
@@ -0,0 +1,57 @@
+(function (global, doc, ibexa) {
```

Comment:
Maybe we don't need to pass window like global and window.document since I don't use them inside iife

Reply:
this is just to be consistent with the whole code base.